package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ajax",
+  "version": "1.0.0",
```

Comment:
This version should be the same version that is on bower.json. And as this is a new feature, but does not break anything, the third number of version must be increased. So 1.0.4 for both.